ext/bin/run-locust-load-tests

```diff
@@ -0,0 +1,64 @@
+set -eo pipefail
+
+usage() {
+    cat <<EOF
+
+Usage: $(basename "$0") [SHA]
+
+  Will run locus tests. There is a first run of 15 minutes is due to the warm-up
+  period for JVM. Only the second run is the actual test run.
+
+EOF
+}
+
+misuse() { usage 1>&2; exit 2; }
+
+sha="6.x"
+current_time=$(date +%Y-%m-%dT%H:%M:%S%z)
+
+git fetch --all
+
+if  [-z "$1"]; then
+    sha=$1
+    git checkout "$sha"
+fi
+
+git status
+git reset --hard origin/"$sha"
+
+tmpdir="/home/andrei.filipovici/performance_results"
+export PATH=$PATH:/home/andrei.filipovici/pg_script
+
+../pdb-script/pdb-script --name 100000 run >"$tmpdir/log.txt" 2>&1 &
+pdb_id=$!
+
+printf "Waiting for PDB startup (log: %q)\n" "$tmpdir/log.txt" 1>&2
+
+while ! grep -Fq "Finished database garbage collection" "$tmpdir/log.txt"; do
+    sleep 1
+    if [ "$(grep -F "Error during service start!!!" "$tmpdir/log.txt")" ]; then
+      echo "PDB failed during start-up." 1>&2
+      exit 2
+    fi
+done
+
+
+printf "Starting locust warm-up"
+# We use the same filename to have them overwritten by the test run.
+python3.6 ./locust/run-load-test -t 1m -u 2 --csv "$tmpdir/${sha}/${current_time}" 2>&1
```

Comment:
run time should be 15 minutes, 20 users and run all tasks

Reply:
Updated. I forgot to replace after a few test runs.